CHANGELOG.rst

```diff
@@ -6,6 +6,27 @@
 Synapse Changelog
 *****************
 
+v2.190.0 - 2024-11-29
```

Comment:
I might be mistaken, but don't you usually delete the change files in these PRs?